utils/ocm.py

```diff
@@ -77,6 +77,47 @@ def _get_cluster_ocm_spec(cluster):
         }
         return ocm_spec
 
+    def create_cluster(self, name, cluster):
+        """
+        Creates a cluster.
+
+        :param name: name of the cluster
+        :param cluster: a dictionary representing a cluster desired state
+
+        :type name: string
+        :type cluster: dict
+        """
+        api = f'/api/clusters_mgmt/v1/clusters'
+        cluster_spec = cluster['spec']
+        cluster_network = cluster['network']
+        ocm_spec = {
+            'name': name,
+            'cloud_provider': {
+                'id': cluster_spec['provider']
+            },
+            'region': {
+                'id': cluster_spec['region']
+            },
+            'multi_az': cluster_spec['multi_az'],
+            'nodes': {
+                'compute': cluster_spec['nodes'],
+                'compute_machine_type': {
+                    'id': cluster_spec['instance_type']
+                }
+            },
+            'storage_quota': {
+                'value': int(cluster_spec['storage'] * pow(1024, 3))
+            },
+            'load_balancer_quota': cluster_spec['load_balancers'],
+            'network': {
+                'machine_cidr': cluster_network['vpc'],
+                'service_cidr': cluster_network['service'],
+                'pod_cidr': cluster_network['pod'],
+            }
+        }
+
+        self._post(api, ocm_spec)
```

Comment:
There's some cases where it would be useful to have more info than just the status code. Could you please also add the response body to the _post function, based on the current log level?

Reply:
Mostly thinking of the following cases:
- Not enough quota on our org
- Wrong spec in app-interface, thus an incorrect payload gets sent to the create API
- The OCM API changes and we're missing parameters (maybe)

Reply:
done!